hazelcast/src/main/java/com/hazelcast/map/impl/event/MapEventPublisherImpl.java

```diff
@@ -265,6 +265,11 @@ public void hintMapEvent(Address caller, String mapName, EntryEventType eventTyp
         // NOP
     }
 
+    @Override
+    public boolean hasEventListener(String mapName) {
+        return eventService.hasEventRegistration(SERVICE_NAME, mapName);
```

Comment:
how about checking if there is a listener which configured with `includeValue=true` ?